charts/nri-kube-events/templates/_helpers.tpl

```diff
@@ -0,0 +1,23 @@
+{{/* vim: set filetype=mustache: */}}
+
+{{- define "nri-kube-events.securityContext.pod" -}}
+{{- if include "newrelic.common.securityContext.pod" . -}}
+{{- include "newrelic.common.securityContext.pod" . -}}
+{{- else if include "newrelic.compatibility.securityContext" . -}}
+{{- include "newrelic.compatibility.securityContext" . -}}
```

Comment:
shouldn't this definition return an string ? something like this 

```
runAsUser: {{ include "newrelic.compatibility.securityContext" . }}
```

Reply:
I think that as we are using `{{ dict "runAsUser" .Values.runAsUser | toYaml }}`  to create the value in the compatibility layer, we are good. However, I removed the right dash.